src/lib/network_pool/transaction_pool.ml

```diff
@@ -628,13 +665,24 @@ struct
                                    transactions at the same nonce to different
                                    nodes, which will then naturally gossip them.
                                 *)
-                                Logger.debug t.logger ~module_:__MODULE__
+                                let f_log =
+                                  if is_sender_local then Logger.error
```

Comment:
did you intend to keep these logs as error messages? I imagine you did this just for them to show up better in the output. Maybe you could add something to the `metadata` field if `is_sender_local` if you think these logs may be useful to trace in the future.

Reply:
These would be error messages for local transactions and debug for gossiped transactions. Since the client command `send-payment` doesn't reflect any of the transaction pool errors, I made this to show up better in the output. The error message has the information needed